docs/vectorizer/README.md

```diff
@@ -0,0 +1,673 @@
+# Automate AI embedding with pgai Vectorizer
+
+Vector embeddings transform text and documents into compact, semantically rich representations that enable context-aware searches beyond traditional keyword matching.
+
+While PostgreSQL excels at storing and querying these embeddings, synchronizing embeddings with source data has traditionally required custom solutions from developers.
+
+pgai Vectorizer provides a SQL-level interface that automates the embedding process within your database. By treating embeddings as a declarative feature—similar to an index—we simplify the entire workflow.
+
+With pgai Vectorizer, you can:
+
+- Designate text columns or documents for embedding using customizable rules
+- Automatically generate and maintain searchable embedding tables 
+- Keep embeddings synchronized with source data (asynchronously)
+- Access a view that joins base tables with their embeddings
+
+This page provides an overview of Vectorizer features. For quick setup instructions, see the 
+[Vectorizer quick start](/docs/vectorizer/quick-start.md). For technical details, see the
+[Vectorizer API reference](/docs/vectorizer/api-reference.md).
+
+Vectorizer uses a background worker for embedding generation, making the process performant and resilient to LLM endpoint failures. On [Timescale Cloud](https://tsdb.co/gh-pgai-signup), the worker runs automatically. For other cloud providers (AWS RDS, Supabase, etc.) or self-hosted Postgres, use the [vectorizer worker](/docs/vectorizer/worker.md) to process your vectorizers.
+
+## How vectorizer works
+
+The vectorizer is designed to be flexible and customizable. Each vectorizer defines a pipeline for creating embeddings from your data. The pipeline consists of these components:
+
+- **[Loading](/docs/vectorizer/api-reference.md#loading-configuration):** Defines the source of data to embed - either directly from a database column or from an external file via URI
+- **[Parsing](/docs/vectorizer/api-reference.md#parsing-configuration):** Converts non-text documents (PDF, HTML, markdown) into a text format suitable for embedding
+- **[Chunking](/docs/vectorizer/api-reference.md#chunking-configuration):** Splits text data into smaller, semantically meaningful pieces
+- **[Formatting](/docs/vectorizer/api-reference.md#formatting-configuration):** Prepares each chunk for embedding, optionally adding context like document title or metadata
+- **[Embedding](/docs/vectorizer/api-reference.md#embedding-configuration):** Generates vector embeddings using your chosen LLM provider and model
+
+Let's explore how the Vectorizer can transform your approach to unstructured data analysis and semantic search:
+
+- [Select an embedding provider and set up your API Keys](#select-an-embedding-provider-and-set-up-your-api-keys)
```

Comment:
I think this table of contents may need to be updated. It's missing the appendix.

Reply:
I believe it can be even removed, GH already provides such a feature. Otherwise, if we still wanna include a TOC, should be generated automatically.